n3fit/src/n3fit/tests/regressions/quickcard_2.json

```
@@ -49,7 +49,7 @@
       "trainable": true
     }
   ],
-  "stop_epoch": 313,
+  "stop_epoch": 314,
   "best_epoch": 11,
```

```suggestion
  "best_epoch": 12,
```
I'm not sure why the introduction of new stuff into the conda recipe changes this but it seems it does.